packages/compass-settings/src/components/settings/general.spec.tsx

```diff
@@ -55,6 +55,16 @@ describe('GeneralSettings', function () {
     });
   });
 
+  it('renders defaultSortOrder', function () {
+    expect(within(container).getByTestId('defaultSortOrder')).to.exist;
+  });
+
+  it('changes defaultSortOrder value when selecting an option', function () {
+    const select = within(container).getByTestId('defaultSortOrder');
+    userEvent.selectOptions(select, '_id: 1 (in ascending order by creation)');
```

Comment:
This test fails for me locally (and in CI):

```
1 failing

  1) GeneralSettings
       changes defaultSortOrder value when selecting an option:
     TestingLibraryElementError: Value "_id: 1 (in ascending order by creation)" not found in options
```

I'm pretty sure `selectOptions` works only with browser select and you'll have to actually click around things to select something in a leafygreen component. Does it work for you?